book/src/checkpoint-sync.md

```diff
@@ -146,8 +146,8 @@ For more information on historic state storage see the
 
 To manually specify a checkpoint use the following two flags:
 
-* `--checkpoint-state`: accepts an SSZ-encoded `BeaconState` blob
-* `--checkpoint-block`: accepts an SSZ-encoded `SignedBeaconBlock` blob
+* `--checkpoint-state`: accepts an SSZ-encoded `BeaconState` file
+* `--checkpoint-block`: accepts an SSZ-encoded `SignedBeaconBlock` file
 
```

Comment:
We also need `--checkpoint-blob` post-Deneb I think.

Reply:
https://github.com/sigp/lighthouse/pull/5252

Reply:
Added in [f7b0af5](https://github.com/sigp/lighthouse/pull/5934/commits/f7b0af541aa606c333bdb28030910c1dc751d6a0)